### DISCUSSION TITLE
Windows-like icons, or real icons?

### DISCUSSION THREAD

**Discussion by PonyRoleplayer** [nestoris/Win98SE#47] [Ideas]
Something that always wondered is whether windows-like icons for apps is better than the real icons from that era. In my case, I see all LibreOffice icons are replaced by Office icons, and while it helps to give a Window-esque vibe, it also kind of lies in a way to the user. Same with Thunderbird that has an outlook icon.

So I want to ask you guys. What do you guys prefer? Windows fake app icons, or period-accurate icons for the apps? 

To give you an example, this is what old thunderbird icon looked like:
![image](https://github.com/user-attachments/assets/7475ca6d-57c7-4b87-9195-957868693745)

**Comment by nestoris** [comment]
You can open a vote on this issue, as it was with FireFox icon, but there is a counter question: who will scale these icons to the size of at least 128x128? They all have their own style, and it's a good idea for a creative approach-to understand the essence and principle, why they are made this way, and what is revealed in them from what is not visible in their small size. I agree that the spirit of the era is a good thing, but when I made this theme initially for myself, I set the goal to make it practical and clear (so that one icon differs from another), as well as esthetic enough that the pixel remains a pixel. It seems to me that you can make some extras so that you can add them to the theme yourself: user unpacked the archive, executed the script where he specified the path to index.theme file and voila: the desired icons are installed.
